core/qqbouchot.cpp

```diff
@@ -740,6 +742,23 @@ void QQBouchot::parsingFinished()
 		m_history.append(m_newPostHistory);
 		m_lastId = m_parser->maxId();
 		m_state.hasNewPosts = true;
+
+		//On vérifie pour chaque post s'il est le seul pour la minute
+		if (m_bSettings.isShortNorlogeEnabled()) {
+			for (int i = std::max(1, m_history.size()-m_newPostHistory.size()-1); i<m_history.size() -1  ; ++i) {
```

Comment:
http://doc.qt.io/qt-5/qtglobal.html#qMaxx Is more Qt way (and prevents new import)

Reply:
Maybe instead of working on m_history, this loop should be run on m_newPostHistory before it is appended on m_history. This also would suppose starting by the last item of m_history (if exists).

Reply:
I'm also not sure that the first post of history won't by treated as alone in minute as it should never (we don't know what's just before)